src/routes/assets/+page.svelte

```diff
@@ -64,22 +66,27 @@
             <div class="web-grid-120-1fr-auto">
                 <header class="web-grid-120-1fr-auto-header">
                     <h1 class="web-display web-u-color-text-primary">Brand assets</h1>
-                    <button
-                        class="u-flex u-width-full-line web-u-padding-block-20
-                        web-u-color-text-primary web-is-only-mobile
-                       web-u-margin-inline-32-negative u-margin-block-start-24 web-u-sep-block u-width-full-line"
-                    >
-                        <span
-                            class="web-container u-flex u-width-full-line u-main-space-between u-cross-center"
+                    <div class="web-is-only-mobile">
+                        <button
+                            class="web-u-padding-block-20
+                            web-u-color-text-primary web-is-only-mobile
+                            u-margin-block-start-24 web-u-sep-block u-padding-inline-20 u-margin-inline-negative-20"
+                            style="inline-size: 100vw;"
+                           on:click={() => (showToc = !showToc)}
                         >
-                            <span class="web-description">Table of contents</span>
-                            <span class="icon-menu-alt-4" aria-hidden="true" />
-                        </span>
-                    </button>
+                            <span
+                                class="u-flex u-main-space-between u-cross-center"
+                                style="inline-size: 100%;"
+                            >
+                                <span class="web-description">Table of contents</span>
+                                <span class="icon-menu-alt-4" aria-hidden="true" />
+                            </span>
+                        </button>
+                    </div>
                 </header>
-                <TocNav />
+                <TocNav bind:showToc />
                 <main class="web-grid-120-1fr-auto-main /web-is-mobile-closed" id="main">
-                    <div class="web-content">
+                    <div class="web-content" class:web-is-mobile-closed={showToc && !showToc}>
```

Comment:
`{showToc && !showToc}` should always evaluate false right?

Reply:
I think we might not even need this class here. This PR still requires further fixes anyways.